README.md

```diff
@@ -44,7 +44,35 @@ crontab -e
 @reboot python -m leeteasy start 14:30 &
 
 ```
-## Contribution guideline
+## Contributing
 
-If you like this project and want to improve by adding features, fixing bugs or anything, please follow
-the [contributing guidelines](docs/CONTRIBUTING.md).
\ No newline at end of file
+We are very happy to see you here. Before sending your pull requests, make sure that you read the whole workflow and the contributing guidelines given below. If you have any doubt regarding the contributing guide, please feel free to [state it clearly in an issue](https://github.com/sudiptob2/leet-easy/issues/new/choose)
+
+
+### Workflow
```

Comment:
I think this workflow part should be moved into `contributing.md`. Can you update ?